### PR TITLE
Doc cleanup: formatting, grammar, consistency, minor fixes

### PR DESCRIPTION
**Pull request by bpaduch**
Clean up of reference docs, particularly: adding reference to common attributes (when appropriate), fix formatting, grammar, links, and overall consistency.

to: @pbakaus

### CODE REVIEW

File: extensions/amp-list/amp-list.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ response and accessed using, e.g. `items="field1.field2"` expression.
 
 Thus, when `items="items"` is specified (the default) the response must be a JSON object that
 contains an array property "items":
-```json
+```text
```
Comment (pbakaus) [inline extensions/amp-list/amp-list.md]: Why was this changed to text?

File: extensions/amp-reach-player/amp-reach-player.md [hunk 2]
```diff
@@ -20,13 +20,12 @@ limitations under the License.
...
     <td width="40%"><strong>Description</strong></td>
-    <td>
-    An <code>amp-reach-player</code> component displays the Reach Player configured in the <a href="http://beachfrontreach.com">Beachfront Reach</a> platform.
+    <td>Displays the Reach Player configured in the <a href="http://beachfrontreach.com">Beachfront Reach</a> platform.
     </td>
   </tr>
   <tr>
     <td width="40%"><strong>Availability</strong></td>
-    <td>Beta</td>
+    <td>Stable</td>
```
Comment (pbakaus) [inline extensions/amp-reach-player/amp-reach-player.md]: think the availability field is sort of redundant when it says "stable". I'd much prefer to remove it all together, and create a special callout for things that are not stable.

File: extensions/amp-ad/amp-ad.md [hunk 3]
```diff
@@ -1,6 +1,8 @@
 # <a name="amp-ad"></a> `amp-ad` / `amp-embed`
 
-NOTE: The specification of `amp-ad` / `amp-embed` is likely to significantly evolve over time. The current approach is designed to bootstrap the format to be able to show ads.
+{% call callout('Note', type='note') %}
```
Comment (bpaduch) [inline extensions/amp-ad/amp-ad.md]: Yes, all callouts in GitHub don't look nice.  I'll make a note about automating callouts in my list of things to do.

File: extensions/amp-ad/amp-ad.md [hunk 4]
```diff
@@ -182,33 +184,38 @@ This is due to the UX implications of full page overlay ads. It may be considere
 
 ## Attributes
 
-### type
+**type**
```
Comment (bpaduch) [inline extensions/amp-ad/amp-ad.md]: I think attributes shouldn't be titles - they render awfully.  Tables would be better, which I might do in the next round of updates - after the agency changes the site.  Actually, in most dev docs, attributes are description list (`<dl>` `<dt>` .. etc.) which looks better but requires a lot of html.  I'm still pondering it, but for now, I felt using bold was the best option.

File: extensions/amp-list/amp-list.md (hunk 1 above)
Comment (bpaduch) [inline extensions/amp-list/amp-list.md]: Pygments renders errors in red, which looks horrible. I felt in this case, nothing was lost by having it defined as text, and it got rid of the annoying red highlighting.  I have that as a discussion point to talk to you about.

File: extensions/amp-carousel/amp-carousel.md [hunk 5]
```diff
@@ -47,7 +47,7 @@ The carousel consists of an arbitrary number of items, as well as optional navig
 
 The carousel advances between items if the user swipes, uses arrow keys, or clicks an optional navigation arrow.
 
-**Example**: While the example shows a carousel of images, `amp-carousel` supports arbitrary children.
+**Example**: While the example shows a carousel of images, `amp-carousel` supports arbitrary HTML children.
```
Comment (bpaduch) [inline extensions/amp-carousel/amp-carousel.md]: I would refrain using callouts for examples, otherwise, our docs will be all callouts. :)

File: extensions/amp-ad/amp-ad.md [hunk 6]
```diff
@@ -218,18 +225,18 @@ Optionally `amp-ad` supports a child element with the `placeholder` attribute. I
 
 ## No Ad available
 - `amp-ad` supports a child element with the `fallback` attribute. If supported by the ad network, this element is shown if no ad is available for this slot.
+- If there is no fallback element available, the amp-ad tag will be collapsed (set to display: none) if the ad sends a message that the ad slot cannot be filled and AMP determines that this operation can be performed without affecting the user's scroll position.
```
Comment (bpaduch) [inline extensions/amp-ad/amp-ad.md]: Done

File: extensions/amp-audio/amp-audio.md [hunk 7]
```diff
@@ -79,6 +79,10 @@ If present, will automatically loop the audio back to the start upon reaching th
 
 If present, will mute the audio by default.
 
+**common attributes**
```
Comment (bpaduch) [inline extensions/amp-audio/amp-audio.md]: See my note above.

File: extensions/amp-reach-player/amp-reach-player.md (hunk 2 above)
Comment (bpaduch) [inline extensions/amp-reach-player/amp-reach-player.md]: Okay.  I'll mark that as a future improvement.  For now, all elements have the availability "field".  I think too that I would like to use this in my future way of tagging - using meta of some sort.

File: extensions/amp-ad/amp-ad.md (hunk 3 above)
Comment (pbakaus) [inline extensions/amp-ad/amp-ad.md]: SGTM.

File: extensions/amp-ad/amp-ad.md (hunk 4 above)
Comment (pbakaus) [inline extensions/amp-ad/amp-ad.md]: OK, agree that bold is probably better than titles. Fine for now, but looking forward to a saner future solution.

File: extensions/amp-carousel/amp-carousel.md (hunk 5 above)
Comment (pbakaus) [inline extensions/amp-carousel/amp-carousel.md]: Sure, wfm.

File: extensions/amp-list/amp-list.md (hunk 1 above)
Comment (pbakaus) [inline extensions/amp-list/amp-list.md]: Got it. Yeah, in this limited example it doesn't hurt much.

File: extensions/amp-reach-player/amp-reach-player.md (hunk 2 above)
Comment (pbakaus) [inline extensions/amp-reach-player/amp-reach-player.md]: OK, fine for now. In general, I'm a fan of keeping our docs as lean as possible, looking forward to brainstorm with you soon.
